job-server/src/main/scala/spark/jobserver/io/DataFileDAO.scala

```diff
@@ -134,16 +135,17 @@ class DataFileDAO(config: Config) {
     false
   }
 
-  private def readFileInfo(in: DataInputStream) = DataFileInfo(in.readUTF, new DateTime(in.readLong))
+  private def readFileInfo(in: DataInputStream) = DataFileInfo(in.readUTF,
+    ZonedDateTime.ofInstant(Instant.ofEpochMilli(in.readLong), ZoneId.systemDefault()))
 
   private def addFile(aName: String) {
     files += aName
   }
 
   def listFiles: Set[String] = files.toSet
 
-  private def createFileName(aName: String, uploadTime: DateTime): String =
-    aName + "-" + uploadTime.toString().replace(':', '_')
+  private def createFileName(aName: String, uploadTime: ZonedDateTime): String =
+    aName + "-" + DateTimeFormatter.ISO_INSTANT.format(uploadTime).replace(':', '_')
```

Comment:
Hm, is this time format the same?  (Wouldn't toString still work the same way?)

Reply:
Good catch. I only checked UTC and for other time zones file names were not identical. Fixed it now. The normal `toString` changed with Java time producing something like `2021-05-06T10_44_20.632275+02_00[Europe/Berlin]`